### DISCUSSION TITLE
ROS2 version?

### DISCUSSION THREAD

**Discussion by ppbrown** [ros-industrial/industrial_core#287] [General]
I'm kinda surprised there is no pointer to a ROS2 version.
Anyone already done the work?
Is it planned?

**Comment by gavanderhoorn** [comment]
See #253.

**Comment by ppbrown** [comment]
Thank you for the reference.
My summary of that 2 year old ticket would be:
- there is no desire to update the existing code to ros 2
- there is dissatisfaction with the current design, so there is a wish to rewrite/modernize the API.. but no commitment or will to do so.
Which to me basically says this repo/subproject is dead :(
